src/precess_cd.jl

```diff
@@ -0,0 +1,75 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _precess_cd{T<:AbstractFloat}(cd::AbstractMatrix{T}, epoch1::T, epoch2::T, crval_old::AbstractVector{T},
+                                       crval_new::AbstractVector{T}, FK4::Bool)
+    t = deg2rad((epoch2 - epoch1)*0.001)
```

Comment:
I think that you shouldn't call `deg2rad` here, `t` isn't simply a global multiplicative factor in definition of `c`, it's rather a third-order polynomial in `t`.